examples/bearriver/src/FRP/BearRiver.hs

```diff
@@ -153,6 +155,21 @@ after q x = feedback q go
                   ct = if t' < 0 then constant (NoEvent, t') else go
               return ((e, t'), ct)
 
+occasionally :: MonadRandom m 
+            => Time -- ^ The time /q/ after which the event should be produced on average
+            -> b    -- ^ Value to produce at time of event
+            -> SF m a (Event b)
+occasionally tAvg b
+  | tAvg <= 0 = error "dunai: Non-positive average interval in occasionally."
+  | otherwise = proc _ -> do
+    r <- getRandomRS (0, 1) -< ()
+    dt <- timeDelta         -< ()
+    let p = 1 - exp (-(dt / tAvg))
+    returnA -< if r < p then Event b else NoEvent
+ where
+  timeDelta :: Monad m => SF m a DTime
```

Comment:
Consistency in indentation?

Reply:
Do you mean the where? I looked at the other code in BearRiver.hs and found that where was indented with 1 space. timeDelta is not as far indented as the above code because of guards.

Reply:
You're right about the `where`. This is my fault. It's one of those places in which I tried to apply an indentation rule and, further down the line, I realised it was no good.

I tend to use two spaces:
```haskell
f x y z =
  someExpression x z y
```
That also applies to `where` clauses:

```haskell
f x y z = someExpression z'
  where
    z' = anotherExpression x y z 
```
The problem appears because I tend to put `where` to the left of the expression that uses it. That means I need to add 4 spaces on top, or only one for `where`:
```
f x y z =
    someExpression z'
  where
    z' = anotherExpression x y z 
```
Or
```
f x y z =
 someExpression z'
where
 z' = anotherExpression x y z 
```
All of which look wrong to my eye :( The second has the added issue that lhs2TeX won't accept it as properly indented haskell, and everything will be aligned to the left as top-level expressions.

So, I'll accept it as is and later do my job at making the syntax more consistent across the whole project, which I haven't done properly so far :|

Thanks!